crates/modelardb_storage/src/metadata/table_metadata_manager.rs

```diff
@@ -1103,6 +1046,23 @@ mod tests {
             .unwrap());
     }
 
```

Comment:
Maybe add a comment documenting which functions are tested as is done elsewhere.